test/lit/passes/dae-gc-refine-params.wast

```diff
@@ -410,13 +414,17 @@
   ;; a direct ref.null because those can be rewritten by LUBFinder.
   (select
    (ref.null none)
-   (struct.new_default $"{i32}")
+   (select
+     (struct.new_default $"{i32}")
+     (struct.new_default $"{i32_i64}")
+     (i32.const 0)
+   )
```

Comment:
perhaps add a comment here? i can't figure out why it was needed.

Reply:
Done.